modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/provider/SystemRelatedModelsFDSDataProvider.java

```diff
@@ -83,35 +89,44 @@ public List<RelatedModel> getItems(
 		long objectEntryId = ParamUtil.getLong(
 			httpServletRequest, "objectEntryId");
 
+		User user = _userLocalService.getUser(PrincipalThreadLocal.getUserId());
+
 		return TransformUtil.transform(
 			(List<BaseModel<?>>)objectRelatedModelsProvider.getRelatedModels(
 				objectScopeProvider.getGroupId(httpServletRequest),
 				objectRelationshipId, objectEntryId, fdsKeywords.getKeywords(),
 				fdsPagination.getStartPosition(),
 				fdsPagination.getEndPosition()),
 			relatedModel -> {
-				String objectFieldDBColumnName =
+				String businessType = null;
+
+				String objectFieldName =
```

Comment:
Why we change from `objectFieldDBColumnName` to `objectFieldName`?

Reply:
The `objectFieldName` seems more clearly for me in this case because `objectFieldDBColumnName` and `objectFieldName` are both object field names.

@carolmariaabb can you help us to define this name?

Reply:
This was changed by [this commit](https://github.com/liferay-objects/liferay-portal/pull/2224/commits/faaf7783b107e8397d3c6a7977d136eeeb1d7289#diff-c7993ce3095db17e0b6a6cc2289c32a8f8363a3945b360acc8e00b35a2a186bc).